README.md

```diff
@@ -404,340 +364,39 @@ Simulation results:
- * `HIVE_INIT_BLOCKS` path to a folder of blocks to import after seeding (default = "/blocks/")
- * `HIVE_INIT_KEYS` path to a folder of account keys to import after init (default = "/keys/")
-
-*Note: It is up to simulators to wire the clients together. The simplest way to do this is to start
-a bootnode inside the simulator and specify it for new clients via the documented `HIVE_BOOTNODE`
-environment variable. This is required to make simulators fully self contained, also enabling much
-more complex networking scenarios not doable with forced fixed topologies.*
-
-The simulation will be considered successful if and only if the exit code of the entrypoint script
-is zero! Any output that the simulator generates will be saved to an appropriate log file in the `hive`
-workspace folder and also echoed out to the console on `--loglevel=6`.
-
-#### Reporting sub-results
-
-It may happen that the setup/teardown cost of a simulation be large enough to warrant validating not
-just one, but perhaps multiple invariants in the same test. Although the results for these subtests
-could be reported in the log file, retrieving them would become unwieldy. As such, `hive` exposes a
-special HTTP endpoint on its RESTful API that can add sub-results to a single simulation. The endpoint
-resides at `/subresults` and has the following parameters:
-
- * `name`: textual name to report for the subtest
- * `success`: boolean flag (`true` or `false`) representing whether the subtest failed
- * `error`: textual details for the reason behind the subtest failing
- * `details`: structured JSON object containing arbitrary extra infos to surface
-
-For example, doing a call to this endpoint with curl:
-
-```
-$ curl -sf -v -X POST -F 'details={"Preconditions failed": ["nonce 1 != 2", "balance 0"]}' \
-  "$HIVE_SIMULATOR/subresults?name=Demo%20error&success=false&error=Something%20went%20wrong..."
-```
-
-will result a `subresults` field
-
-```json
-"subresults": [
-  {
-    "name": "Demo error",
-    "success": false,
-    "error": "Something went wrong...",
-    "details": {
-      "Preconditions failed": [
-        "nonce 1 != 2",
-        "balance 0"
-      ]
-    }
-  }
-]
+```bash
+hivechain generate -genesis <path to genesis file> -length <desired length of chain>
 ```
-### Closing notes
-
- * There is no constraint on how much time a simulation may run, but please be considerate.
- * The simulator doesn't have to terminate nodes itself, upon exit all resources are reclaimed.
-
-# Generating test blockchains
-
-`Hive` allows you to create rlp encoded blockchains for inclusion into sync tests 
-
-## To generate a blockchain offline
-
-`Hive` can be run with the `chainGenerate` setting to generate a blockchain according to specification
-and then exit. The current version only generates blocks with empty transactions, but this will be
+The current version only generates blocks with empty transactions, but this will be
```

Comment:
This is not true, hivechain will generate transactions if certain accounts are present in the genesis block. Check source for the account addresses.